poetry/console/commands/new.py

```diff
@@ -4,6 +4,8 @@
 from cleo import option
 
 from poetry.utils.helpers import module_name
+from poetry.vcs.git import GitConfig
```

Comment:
```suggestion
from poetry.core.vcs.git import GitConfig
```

Reply:
This results in an `ImportError` for me

Reply:
https://www.github.com/python-poetry/poetry-core/tree/master/poetry%2Fcore%2Fvcs%2Fgit.py

Reply:
Yes, sorry about that. This branch was pretty outdated, I needed to merge from `upstream/master`. Thanks!